fs/dafsserver/dafsserver.cpp

```diff
@@ -2236,7 +2236,7 @@ class CRemoteIndexBaseActivity : public CRemoteDiskBaseActivity
         crc32.tally(sizeof(time_t), &modTimeTT);
         unsigned crc = crc32.get();
 
-        keyIndex.setown(createKeyIndex(fileName, crc, isTlk));
+        keyIndex.setown(createKeyIndex(fileName, crc, isTlk, 0));
```

Comment:
future PR? we should pick up blockedIndex size here for plane involve here in dafilesrv context too.